_CodingChallenges/114-bubble-sort.md

```diff
@@ -9,6 +9,13 @@ live_example: false
 links:
   - title: "Visualizing Algorithms"
     url: "https://bost.ocks.org/mike/algorithms/"
+    
```

Comment:
The tests fail because there's some whitespace here. Here's a quick fix to remove it:
```suggestion

```